subsys/bluetooth/controller/ll_sw/isoal.c

```diff
@@ -28,6 +28,23 @@
 #include "common/log.h"
 #include "hal/debug.h"
 
+/* Minimum number of bytes that would make inserting a new segment into a PDU
+ * worthwhile.
+ */
+#if defined(CONFIG_BT_CTLR_ISO_TX_SEG_PLAYLOAD_MIN)
+#define ISOAL_SEGMENT_MIN_PAYLOAD         CONFIG_BT_CTLR_ISO_TX_SEG_PLAYLOAD_MIN
+#else
+#define ISOAL_SEGMENT_MIN_PAYLOAD         (1)
+#endif /* CONFIG_BT_CTLR_ISO_TX_SEG_PLAYLOAD_MIN */
```

Comment:
I'm curious:
Why is a TX Kconfig used here? 
And in what case is `CONFIG_BT_CTLR_ISO_TX_SEG_PLAYLOAD_MIN` not defined where we have ISO-AL?

Reply:
I think your observation is mostly correct. If the ISO-AL supports TX then it should be that `CONFIG_BT_CTLR_ISO_TX_SEG_PLAYLOAD_MIN` is defined with some value. So it should be possible to rely on the default.

Reply:
But is `ISOAL_SEGMENT_MIN_PAYLOAD` only used for TX?

Reply:
Yes it is only used for TX. That is the threshold that would make inserting a new segment worthwhile when packing PDUs.

Reply:
OK. The name of the macro `ISOAL_SEGMENT_MIN_PAYLOAD` doesn't really explicitely state this, hence my confusion. 

If `ISOAL_SEGMENT_MIN_PAYLOAD` is for TX only, this is fine, although the name of the macro may need to more explicit perhaps. 

But if `ISOAL_SEGMENT_MIN_PAYLOAD` is for TX only, then why do we need this macro at all? If we can TX in the ISO-AL, wouldn't `CONFIG_BT_CTLR_ISO_TX_SEG_PLAYLOAD_MIN` always be defined?

Reply:
Sorry about the omission. I have updated it now. Yes it's not strictly necessary. It was a remnant of how it evolved.